regex and realese notes chaned (#15547)
demisto/content: Packs/CommonTypes/ReleaseNotes/3_1_21.md

#### Indicator Types

> - **domainRepUnified** - Extract domains using regex in order to catch non-ASCII characters, extract domains/hostnames from provided domains/hostnames, URLs and email addresses and supports both escaped and unescaped domains.

Reviewer: ```suggestion
- **domainRepUnified** - Updated the regex in order to support extraction of domains/hostnames with non-ASCII characters or escaped characters.
```
@melamedbn correct me if I'm wrong, "extract domains/hostnames from provided domains/hostnames, URLs and email addresses" + "unescaped domains" are something we already support (except for the explicit cases above). if not - would appreciate a clarification.
